assets/questions/pirateTranslator.js

```diff
@@ -224,6 +273,61 @@ globalData.questions['pirateTranslator'] = {  // eslint-disable-line dot-notatio
           "Don't forget spaces in your output."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysPrependSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'beginning of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.alwaysAppendSpace',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between words, not at the ',
+          'end of the sentence.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.onlySpaceAfterPirateWords',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'Spaces should only appear between each whether English ',
+          'or Pirate.'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.outputingStringIndexes',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'You need to output words, not word indexes. When you ',
+          'call "range(len(list))" it returns numbers 1 through ',
```

Comment:
... it returns the numbers 0 through N -1 (where N is the length of the list), not the items in the list.